libraries/SrcWrapper/src/stm32/stm32_eeprom.c

```diff
@@ -210,6 +239,22 @@ void eeprom_buffer_flush(void)
   memcpy((uint8_t *)(FLASH_BASE_ADDRESS), eeprom_buffer, E2END + 1);
 }
 
+#elif defined(DATA_EEPROM_BASE)
+/**
+  * @brief  This function writes the buffer content into the eeprom on L0
+  * @param  none
+  * @retval none
+  */
+void eeprom_buffer_flush(void)
+{
+  uint32_t i;
+
+  for (i = DATA_EEPROM_BASE ; i <= DATA_EEPROM_END ; i++) {
+    HAL_FLASHEx_DATAEEPROM_Unlock();
```

Comment:
Why not test like in `eeprom_write_byte`?

```suggestion
     if (HAL_FLASHEx_DATAEEPROM_Unlock() == HAL_OK) { 
```

Reply:
More simply: Lock EEPROM then loop writing each byte and unlock